### DISCUSSION TITLE
MAG X570 TOMAHAWK WIFI

### DISCUSSION THREAD

**Discussion by miroresh** [ixjf/MSIRGB#189] [General]
https://www.msi.com/Motherboard/MAG-X570-TOMAHAWK-WIFI 

Any future plans for this bad boy?  It's currently the most recommended MoBo for 5000 series. :)

**Comment by ixjf** [comment]
Not right now. I can't support a motherboard I do not have access to. The list of motherboards supported by MSIRGB at this moment are all those that I can guarantee work the same way as mine, so I test on mine and know that it will work on these others as well.
